pkg/snapshot/filesystem.go

```diff
@@ -719,8 +750,10 @@ func fileSystemMinioConfigPod(clientset kubernetes.Interface, deployOptions File
 		}
 	}
 
-	kotsadmTag := kotsadmversion.KotsadmTag(kotsadmtypes.KotsadmOptions{}) // default tag
-	image := fmt.Sprintf("kotsadm/kotsadm:%s", kotsadmTag)
+	// kotsadmTag := kotsadmversion.KotsadmTag(kotsadmtypes.KotsadmOptions{}) // default tag
+	// image := fmt.Sprintf("kotsadm/kotsadm:%s", kotsadmTag)
+	// TODO: REMOVE ME
+	image := "localhost:32000/kotsadm:v1.49.0-10-gaff83f7e8-dirty@sha256:ad62a01b7b076883c538f1480242bdf57fc79100cda306a241e12171c453ae07"
```

Comment:
I'll change this when I'm done testing - this is how I am using the new check script that looks for writable permissions.